react/react2/inclass/app/blogs/[blogId]/page.jsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+import { usePathname, useSearchParams } from "next/navigation";
+
+export default function blog({ params }) {
```

Comment:
It is generally considered as a bad practice to give your components name starting with the lowercased letter. It should be `Blog` instead of `blog`